cli/run_parser.py

```diff
@@ -177,8 +177,7 @@ def main(
         else:
             try:
                 tasks.append(ParserInput.parse_raw(path.read_text()))  # type: ignore
-
-            except pydantic.error_wrappers.ValidationError as e:
+            except (pydantic.error_wrappers.ValidationError, KeyError) as e:
```

Comment:
Why is the key error thrown?

Reply:
The type for ParserInput has document_content_type optional. Therefore, it would appear that there are outputs from the ingest job that don't have the field document_content_type. This doesn't actually make sense as in the types.py of the ingester a null value is always at least provided for the field.

Reply:
Without knowing which documents are causing the issue it's hard to debug.